src/visualization/utils/useVisDomainSettings.ts

```diff
@@ -211,7 +214,7 @@ export const useZoomRequeryXDomainSettings = (args: ZoomRequeryArgs) => {
 
   const [windowPeriod, setWindowPeriod] = useState<number>(
     normalizeWindowPeriodForZoomRequery(
-      getWindowPeriodFromVariables(query, variables),
+      getWindowPeriodFromVariables(queries[activeQueryIndex], variables),
```

Comment:
`queries[activeQueryIndex]` is returning undefined when selecting time range in `/usage` page, causing the graph to crash. I wonder why that is...🤔

Reply:
ah good catch. i'll check it out.

Reply:
This was an excellent find. So good, I have to mention it again! A bug was found that i didn't even realize existed.

I have pushed up a change to disable Adaptive Zoom whenever queries cannot be found, such as in the Usage page where the data comes from an api rather than a query.

Thanks again @blegesse-git !

Reply:
so glad I could be of help!!